pushapkscript/exceptions.py

```diff
@@ -5,3 +5,7 @@
 class SignatureError(ScriptWorkerTaskException):
     def __init__(self, msg):
         super().__init__(msg, exit_code=STATUSES['internal-error'])
+
+
+class ConfigValidationError(ScriptWorkerTaskException):
+    pass
```

Comment:
I'm assuming I don't need the custom `__init__` method as well here (like `SignatureError`), do I?

Reply:
By default `ScriptWorkerTaskException` exits [with status code 1](https://github.com/mozilla-releng/scriptworker/blob/981e971b710b64fbcef450a7eb1d3c0258999254/scriptworker/exceptions.py#L60) which is considered as [a regular failure](https://github.com/mozilla-releng/scriptworker/blob/be5ca9647f40c9ea9910e578c63293ee9990f8a1/scriptworker/constants.py#L17) by taskcluster. `ConfigValidationError` shows an error on the worker-side, and not on the payload one. Therefore, I would use `STATUSES['internal-error']` to reflect this state. This means we need to overload `__init__()` the same way `SignatureError` does